cgroups/fs/freezer.go

```diff
@@ -41,6 +42,8 @@ func (s *FreezerGroup) Set(path string, cgroup *configs.Cgroup) error {
 			}
 			time.Sleep(1 * time.Millisecond)
 		}
+	default:
+		fmt.Errorf("Invalid argument %s to freezer.state", cgroup.Freezer)
 	}
 
 	return nil
```

Comment:
Hi LK4D4, Thanks for reviewing.
It seems it is.
But what I mainly want to do is to return error clearly when people pass  invalid argument accidentally.
I added new line testing configs.Undefined. How do you think about it?